content/documentation/pages/6-recipes/5-kinesis/1-simple-producer-consumer.md

```diff
@@ -0,0 +1,165 @@
+---
+path: 'recipes/kinesis/simple-producer-consumer/'
+title: 'Amazon Kinesis Binder'
+description: 'A sample of Spring Cloud Stream + Amazon Kinesis Binder in action'
+---
+
+# Amazon Kinesis Sample
+
+The Spring team currently maintains the Spring Cloud Stream Kinesis binder with the help of community contributions.
+You can read more about the binder implementation at [spring-cloud/spring-cloud-stream-binder-aws-kinesis](https://github.com/spring-cloud/spring-cloud-stream-binder-aws-kinesis).
+
+In this walk-through, though, we will review a simple use-case to showcase how the Kinesis binder can be used with Spring Cloud Stream.
+
+## Prerequisite
+
+The only requirement for the demonstration is the "Access Key", "Secret Key", and "Region" credentials, which can be gathered from your AWS account.
+
+## Applications
+
+As for the sample producer and consumer applications, you can clone the repository at [sabbyanandan/kinesisdemo](https://github.com/sabbyanandan/kinesisdemo) to follow along with the sample walk-through.
```

Comment:
Is it really acceptable to link in the official docs your own GitHub repo?
Can't that demo go somewhere in the common one, e.g. https://github.com/spring-cloud/spring-cloud-stream-samples ?

Reply:
I will have to figure out how to do that thing.

Reply:
All good, but this.
Would be great to move it into our common sample. Isn't it?

Reply:
Yep! Agreed. It bugs me also. Let me figure it out when I get to the office.

Reply:
This is now attempted to be ported over to Microsite's samples repo at: https://github.com/spring-cloud/spring-cloud-dataflow-samples/pull/106